modules/data.land/R/soil_process.R

```diff
@@ -11,12 +11,13 @@
 #' @examples
 soil_process <- function(settings, input, dbfiles, overwrite = FALSE,run.local=TRUE){
   
-  if(input$soil$source=="PalEON_soil" && is.null(input$id)){
+
+  if(input$source=="PalEON_soil" && is.null(input$id)){
     PEcAn.logger::logger.severe("currently soil_process requires an input ID to be specified")
     return(NULL)
   }
   
-  if(is.null(input$soil$source)){
+  if(is.null(input$source)){
     input$soil$source <- "PalEON_soil"  ## temporarily hardcoding in the only source
```

Comment:
At this point, I wonder if it makes more sense to have gSSURGO be the new default, or to have this now throw an error if the source isn't specified.